PlantPal/PlantPal/WelcomeScreen.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  ContentView.swift
+//  PlantPal
+//
+//  Created by Marcin Dytko on 07/07/2024.
+//
+
+import SwiftUI
+// MARK: - VIEW
+
+struct WelcomeScreen: View {
+    // MARK: - Properties
+    
+    @EnvironmentObject private var viewModel: WelcomeScreenViewModel
+    
+    // MARK: - BODY
+    var body: some View {
+        NavigationStack {
+            ZStack {
+                
+                Color("BackgroundColor")
```

Comment:
Jako, że wszystkie ekrany będą miały ten sam background color fajnie by było stworzyć jakiś reużywalny komponent, który będzie nam to tło ustawiał. Pomyśl o tym, żeby stworzyć coś co będziemy mogli użyć w ten sposób:
```
MainStack {
   VStack {
       // jakiś dalszy kod w widoku
   }
}
```

Reply:
Dodatkowo zamiast używać stringa, możesz tam użyć 
```
Color(.background)
```
Bardziej czytelne i nie ma szans na literówkę.